### DISCUSSION TITLE
Prośba o możliwość uzyskania po API tekstowej wartości kodu kreskowego z etykiety UPS - Allegro One

### DISCUSSION THREAD

**Discussion by tomasz-grzesik** [allegro/allegro-api#7472] [Suggestions - resolved]
Więcej:
https://github.com/allegro/allegro-api/issues/5695

**Comment by mkawczynski** [comment]
Ja to pobieram szukając bezpośrednio w ZPL-u

**Comment by MartaNowaczyk** [comment]
Wdrożyliśmy możliwość zwracania takiej informacji [w nowym WZA](https://developer.allegro.pl/documentation/#operation/getShipmentDetails), w obiekcie additionalProperties.

**Comment by tomasz-grzesik** [comment]
Dziękuję za wdrożenie i info.
